Dockerfile

```
@@ -182,7 +182,8 @@ EOF
 # ARG PATH=/opt/python/bin:$PATH
 ADD ./requirements.txt /tmp/
 ADD ./voicevox_engine /opt/voicevox_engine/voicevox_engine
-ADD ./run.py ./check_tts.py ./VERSION.txt ./LICENSE ./LGPL_LICENSE /opt/voicevox_engine/
+ADD ./docs /opt/voicevox_engine/docs
+ADD ./run.py ./generate_licenses.py ./check_tts.py ./VERSION.txt ./LICENSE ./LGPL_LICENSE /opt/voicevox_engine/
```

今気づいたのですが、出来上がる完成物は厳密には`./LICENSE`と`./LGPL_LICENSE`が適用されない（ソースコードには適用できるけどcoreはそうではない）ので、誤解が起きづらいように省いちゃいます。

```suggestion
ADD ./run.py ./generate_licenses.py ./check_tts.py ./VERSION.txt /opt/voicevox_engine/
```